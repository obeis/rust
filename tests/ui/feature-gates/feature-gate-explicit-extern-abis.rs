#![feature(explicit_extern_abis)]

extern fn foo() {}
//~^ ERROR extern declarations without an explicit ABI are disallowed

fn main() {}
