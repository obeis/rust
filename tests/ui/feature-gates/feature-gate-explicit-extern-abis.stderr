error: extern declarations without an explicit ABI are disallowed
  --> $DIR/feature-gate-explicit-extern-abis.rs:3:1
   |
LL | extern fn foo() {}
   | ^^^^^^

error: aborting due to 1 previous error

